docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Grafana Mimir version 2.1 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Below we highlight the top features, enhancements and bugfixes in this release, as well as relevant callouts for those upgrading from Grafana Mimir 2.1.
+The complete list of changes is recorded in the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir introduces new experimental support for ingesting out-of-order samples. Feature is enabled by configuring the time window since the most-recently ingested sample for the series. This can be enabled on per-tenant basis.
+
+- **Error messages**: We have overhauled the error messages reported to end user to make them easier to understand, and to include error codes that can be easily searched for. This effort will continue in next releases as well.
+
+- **Prefixed bucket store**: It is now possible to store blocks, rules and alerts into a single bucket, by configuring bucket prefix for each feature.
```

Comment:
```suggestion
- **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, each under its own user-defined prefix, rather than requiring one bucket for each.
```

Reply:
Curious - as a GEM customer, can I set a bucket prefix for the admin-api storage? (Wouldn't be in scope for these release notes, but we'll want to make this callout in the GEM release notes if this is the case). 

Can we point to the section in the configuration this is? or at least mention the name of the right configuration flag?

Reply:
> Curious - as a GEM customer, can I set a bucket prefix for the admin-api storage?

@dimitarvdimitrov may know the answer.